### PR TITLE
chore: add nightly test on more devices

### PR DESCRIPTION
**Pull request by DanielHabenicht**

### CODE REVIEW

File: .github/workflows/nightly-test.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
+name: PR-Checks
```
Comment (DanielHabenicht) [inline .github/workflows/nightly-test.yaml:1]:
```suggestion
name: Nightly-Test
```

File: .github/workflows/nightly-test.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    # https://markus.oberlehner.net/blog/dealing-with-the-browserstack-parallel-test-limit-when-using-testcafe/ http-server
+    # https://github.com/DevExpress/testcafe/issues/1330 Browser List
+    # https://www.npmjs.com/package/testcafe-browser-provider-browserstack testcafe doc
+    - name: test chrome
+      run: |
+        export BROWSERSTACK_USE_AUTOMATE="1"
+        export BROWSERSTACK_DISPLAY_RESOLUTION="1024x768"
+        yarn run http-server dist/ -p 8080 -s &
+        sleep 5
+        curl http://localhost:8080
```
Comment (DanielHabenicht) [inline .github/workflows/nightly-test.yaml:101]:
```suggestion
```

File: .github/workflows/nightly-test.yaml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        yarn run testcafe -c 1 "browserstack:Samsung Galaxy S8" testcafe/*.spec.ts
+        yarn run testcafe -c 1 "browserstack:chrome:Windows 10" testcafe/*.spec.ts
+        yarn run testcafe -c 1 "browserstack:firefox:Windows 10" testcafe/*.spec.ts
+        yarn run testcafe -c 1 "browserstack:edge:Windows 10" testcafe/*.spec.ts
+        yarn run testcafe -c 1 "browserstack:opera:Windows 10" testcafe/*.spec.ts
+        yarn run testcafe -c 1 "browserstack:IE:Windows 10" testcafe/*.spec.ts
+        yarn run testcafe -c 1 "browserstack:safari:OS X Catalina" testcafe/*.spec.ts --skip-js-errors
+        yarn run testcafe -c 1 "browserstack:Pixel 4" testcafe/*.spec.ts
+        yarn run testcafe -c 1 "browserstack:Redmi Note 8" testcafe/*.spec.ts
+        yarn run testcafe -c 1 "browserstack:Moto G7 Play" testcafe/*.spec.ts
```
Comment (DanielHabenicht) [inline .github/workflows/nightly-test.yaml:111]:
```suggestion
        yarn run testcafe -c 1 "browserstack:Moto G7 Play" testcafe/*.spec.ts
        yarn run testcafe -c 1 "browserstack:iPhone XS" testcafe/*.spec.ts
        yarn run testcafe -c 1 "browserstack:iPad Pro 12.9 2020" testcafe/*.spec.ts
```

**Review by ChristophWersal — APPROVED** [review]
I cannot test the changes I guess, but since it is a separate workflow, I would say: go for it.

**Pull request merged**